### PR TITLE
Translate manuals/2.0/ja/ some headings

### PR DESCRIPTION
**Pull request by kenjis**
* Translate some headings
* Fix a previous_page

### CODE REVIEW

File: manuals/2.0/ja/setup.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 layout: docs2-ja
 title: Setting up your virtual host
 permalink: /manuals/2.0/ja/setup/
-previous_page: Models
+previous_page: Command line / console
```
Comment (harikt) [inline manuals/2.0/ja/setup.md]: may be these also ?

Comment (kenjis) [inline manuals/2.0/ja/setup.md]: Yes.

See https://raw.githubusercontent.com/auraphp/auraphp.github.com/master/manuals/2.0/en/setup.md and the next line `previous_page_url`.

`Models` is simply wrong.

Comment (harikt) [inline manuals/2.0/ja/setup.md]: @kenjis , what I was asking you was translating the title and previous_page . 

Thanks!

Comment (kenjis) [inline manuals/2.0/ja/setup.md]: Oh, I see.

Is there no problem to translate `title` and `previous_page`? I'm not sure.

Comment (harikt) [inline manuals/2.0/ja/setup.md]: Seems good to translate to me :) .

**Pull request merged**